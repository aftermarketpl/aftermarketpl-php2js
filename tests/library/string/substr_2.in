$a = "aBcD";
return substr($a, 2, 1);
