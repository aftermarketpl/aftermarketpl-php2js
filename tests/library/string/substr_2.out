var a;
a = "aBcD";
return (a).charAt(2);
